grading/load_config_json.cpp

```diff
@@ -211,7 +217,7 @@ void AddDockerConfiguration(nlohmann::json &whole_config) {
 
       if(this_testcase["containers"][container_num]["container_image"].is_null()){
         //TODO: store the default system image somewhere and fill it in here.
-        this_testcase["containers"][container_num]["container_image"] = "ubuntu:custom";
```

Comment:
where does ubuntu:custom come from now?

Reply:
It comes from the assignment's default container. If there are no defaults set, the default field is populated as being ubuntu:custom on line 126.